modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -734,7 +733,7 @@ public Response getMetadataCacheDetails() {
         }
 
         // Only include `assignment` property if there is a definitive assignment, either to a bucket or not
-        if (assignment.getStatus() != EXPERIMENT_EXPIRED) {
+        if (assignment.getStatus().isDefinitiveAssignment()) {
```

Comment:
@AndreaSuckro can you check with @calvinwma  about this conditional change as this change is not backward compatible especially for PAUSED experiments...